openapi/paths/getters/getState.json

```diff
@@ -68,6 +88,16 @@
                   "type": "integer",
                   "format": "int64"
                 },
+                "totalBurntNQT": {
```

Comment:
Actually these are always included, as they are not expensive to get.

Reply:
oops... thought I removed them. thanks